src/proper_statem.erl

```diff
@@ -218,7 +218,7 @@
 
 -module(proper_statem).
 
--export([commands/1, commands/2, parallel_commands/1, parallel_commands/2,
+-export([commands/1, commands/2, commands/4, parallel_commands/1, parallel_commands/2,
```

Comment:
Where is this used by the "targeted" part of PropEr?

Reply:
I should read my diffs more carefully (or maybe don't make pull requests that late). Thanks for pointing it out.